k2hb_ec2_asg_audit_london.tf

```diff
@@ -136,6 +136,12 @@ resource "aws_launch_template" "k2hb_audit_london" {
 
     tags = local.k2hb_audit_london_tags_asg
   }
+
+  tag_specifications {
+    resource_type = "volume"
+
```

Comment:
Why an empty line here and in other tag_specifications?

Reply:
Just copied the one above it...